README.md

```diff
@@ -99,7 +99,7 @@ Apache Airflow is tested with:
 
 |            | Main version (dev)         | Stable version (2.10.2)    |
 |------------|----------------------------|----------------------------|
-| Python     | 3.8, 3.9, 3.10, 3.11, 3.12 | 3.8, 3.9, 3.10, 3.11, 3.12 |
+| Python     | 3.9, 3.10, 3.11, 3.12      | 3.9, 3.10, 3.11, 3.12      |
```

Comment:
2.10.2 still uses "3.8" min version - and the whole 2.* line should continue doing so.

Reply:
So no backport and dropping support in 2.10.3? Will this contiunue to support 3.8?

Reply:
Reverted.